src/smif/cli/__init__.py

```diff
@@ -435,30 +435,38 @@ def execute_model_run(args):
     ----------
     args
     """
-    timestamp = datetime.datetime.fromtimestamp(time.time()).strftime('%Y%m%dT%H%M%S')
-
     LOGGER.info("Loading resolution data")
     load_resolution_sets(args.directory)
 
-    LOGGER.info("Getting model run definition")
-    model_run_config = get_model_run_definition(args.directory, args.modelrun)
+    if args.batchfile:
+        with open(args.modelrun, 'r') as f:
+            model_runs = f.read().splitlines()
```

Comment:
Can we fail fast here with an error if any of the model runs don't exist?

Reply:
Done